### ISSUE TITLE
setup wizard "advanced"

### ISSUE THREAD

**Issue by gperciva** [Tarsnap/tarsnap-gui#9]
Is it necessary to have an "advanced" option on the first page of the setup wizard?  I think it unnecessarily complicates the process:
- no information is given about which step(s) are "advanced".   (by experimentation, it looks like the "advanced" step is the PATH-selection)
  - if I'm correct, then the "advanced" step will generally be filled-in automatically (if the CLI is in the path).  If the CLI cannot be found, then I think it's important to issue a warning.  Either way, a "beginner" user should not skip this step!
  - I see that if you don't have the CLI and do not select "advanced", then you are automatically taken into the "select the CLI location" screen.  In GUI terms, the top "4 steps" silently turns into "5 steps".  This is unnecessarily confusing.

Basically, I suggest that the "advanced" should be always enabled, and the option should be removed from the title page.

**Comment by shinnok** [comment]
Closed for now.
